js/component/validate.js

```diff
@@ -0,0 +1,43 @@
+import toast from '../component/toast.js';
+export default function validateForm(form) {
+    var validEmailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+    var validPhoneRegex = /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im;
+    if(form.name.value === "") {
```

Comment:
```suggestion
    if(!form.name.value.trim()) {
```
Vì "" là falsy rồi, nên chỉ cần check như này là đủ rồi nhé